### PR TITLE
Session Object Class

### PR DESCRIPTION
**Pull request by betrevisan**
The changes proposed in this PR create the Session object class, which is responsible for handling information regarding a scheduled tutoring session. It also adds the scheduledSessions property to the Tutor object class, which is an array of Session and keeps track of all the sessions scheduled with that specific tutor.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -23,6 +23,13 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+
+    <dependency>
```
Comment (kashiish) [inline pom.xml:27]: I think if you pull from master, this dependency should already be there.

File: src/main/java/com/google/sps/data/Session.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+public final class Session {
```
Comment (dieseldesai) [inline src/main/java/com/google/sps/data/Session.java:17]: I'd suggest naming this something different as "Session" has a specific meaning in terms of HTTP and web which might be confusing here.

Maybe "TutorSession" or "Event"? Also please add javadocs as described here:
https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required

File: pom.xml (hunk 1 above)
Comment (betrevisan) [reply pom.xml:27]: Oh, true! I'll do that :)

File: src/main/java/com/google/sps/data/Session.java (hunk 2 above)
Comment (betrevisan) [reply src/main/java/com/google/sps/data/Session.java]: That makes a lot of sense. I pushed new commits to this PR that address that.

**Comment by betrevisan** [comment]
@googlebot I fixed it.

**Comment by betrevisan** [comment]
@googlebot I fixed it.

**Pull request closed without merging**